### DISCUSSION TITLE
NSC and k8s

### DISCUSSION THREAD

**Discussion by dmytrum** [nats-io/nsc#676] [Q&A] [answered]
Is there any examples or recommendations on how to use NSC for nats in k8s?

**Comment by mixicz** [comment, marked as answer]
> Is there any examples or recommendations on how to use NSC for nats in k8s?

I'm not sure what is your particular use case or problems you encountered. 

`nsc` itself is agnostic of where nats is running, all you need is accessible URL for nats, so all `nsc` examples are valid for nats in k8s. 

If nsc is supposed to run in k8s (e.g. in nats-box), I would not recommend doing it. It would be rather complicated, as you need some writable encrypted storage for private keys.
